myapp/spec/system/tasks_spec.rb

```diff
@@ -3,68 +3,115 @@
 require "rails_helper"
 
 RSpec.describe "Tasks", type: :system do
-  
-  # Create dummy data
-  let!(:tasks) do
-    (1..9).map do |i|
-      Task.create(title: "test title #{i}", description: "test description #{i}", created_at: i.days.ago)
+  describe "test with dummy data" do
+    # Create dummy data
+    let(:tasks) do
```

Comment:
```suggestion
    let!(:tasks) do
```

こちらが定義されている箇所で利用するのであれば let! で良いと思います。
https://zenn.dev/kyohei_shibuya/articles/722891a1a6648a#%E6%9C%AC%E9%A1%8C

Reply:
let!にしました